### PR TITLE
Source file: remove nested properties

### LINKED ISSUE #13787: Source File throwAssertionError: Mismatched number of tables during normalization
- Is this your first time deploying Airbyte?: No
- OS Version / Instance: Ubuntu 18.04
- Memory / Disk: 160Gb / 1 Tb
- Deployment: Docker
- Airbyte Version: 0.35.30-alpha
- Source name/version: File / 0.2.9
- Destination name/version: Postgres / 0.4.6
- Step: The issue is happening during sync
- Description: I faced this issue

https://discuss.airbyte.io/t/issue-with-file-destination-postgres-sync/1331/7
[logs(2).log](https://github.com/airbytehq/airbyte/files/8906381/logs.2.log)

### PR DESCRIPTION
**Pull request by davydov-d**
## What
File source generates invalid schema for JSON format - it contains an excessive empty `properties` layer

## How
Remove redundant layer

## Before
```
{
"streams": [
  {
    "stream": {
      "name": "us_p0_json_18",
      "json_schema": {
        "type": "object",
        "$schema": "http://json-schema.org/draft-07/schema#",
        "properties": {
          "type": "object",
          "$schema": "http://json-schema.org/schema#",
          "required": [
            "data",
            "meta"
          ],
          "properties": {
            "data": {
              "type": "array",
              "items": {
                "type": "array",
                "items": {
                  "type": [
                    "integer",
                    "null",
                    "string"
                  ]
                }
              }
            },
            "meta": {
              "type": "object",
              "required": [
                "view"
              ],
              "properties": {
                "view": {
                  "type": "object",
                  "required": [
                    "approvals",
                    "assetType",
                    "attribution",
                    "attributionLink",
                    "averageRating",~
                   ...
```

## After
```
{
"streams": [
  {
    "stream": {
      "name": "us_p0_json_18",
      "json_schema": {
        "type": "object",
        "$schema": "http://json-schema.org/draft-07/schema#",
        "required": [
            "data",
            "meta"
        ],
        "properties": {
            "data": {
              "type": "array",
              "items": {
                "type": "array",
                "items": {
                  "type": [
                    "integer",
                    "null",
                    "string"
                  ]
                }
              }
            },
            "meta": {
              "type": "object",
              "required": [
                "view"
              ],
              "properties": {
                "view": {
                  "type": "object",
                  "required": [
                    "approvals",
                    "assetType",
                    "attribution",
                    "attributionLink",
                    "averageRating",
```

### CODE REVIEW

File: airbyte-integrations/connectors/source-file/source_file/client.py [hunk 1]
```diff
@@ -248,9 +248,7 @@ def load_nested_json_schema(self, fp) -> dict:
             builder.add_object(json.load(fp))
 
         result = builder.to_schema()
-        if "items" in result and "properties" in result["items"]:
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-file/source_file/client.py:251]: if we ever had a structure like this, it would fail earlier, on line 248

Comment (lazebnyi) [reply airbyte-integrations/connectors/source-file/source_file/client.py:251]: let's add unit test for those case

Comment (davydov-d) [reply airbyte-integrations/connectors/source-file/source_file/client.py:251]: well, I was wrong :( reverted this change
But I also noticed we do not pass required property in the schema. So, now for json and jsonl formats we do not extract properties, but use generated schema as is, only replacing the $schema value. Unit tests added

**Review by lazebnyi — APPROVED** [review]
LGTM

**Pull request merged**